Kitodo/src/main/java/de/sub/goobi/helper/BatchStepHelper.java

```diff
@@ -343,9 +290,14 @@ private void reportProblem() {
      */
     public String solveProblemForSingle() {
         try {
-            solveProblem();
+            serviceManager.getWorkflowService().setSolution(getSolution());
+            try {
+                setCurrentStep(serviceManager.getWorkflowService().solveProblem(this.currentStep, this.solutionTask));
+            } catch (DataException e) {
+                logger.error("Problem couldn't be solved: " + e);
```

Comment:
2 things:
1) If a `DataException` is thrown is it a good idea to go on?
2) Did this kind of error logging work with casting exception object to string message?

Reply:
Look previous.